azure_monitor/src/azure_monitor/utils.py

```diff
@@ -88,7 +99,6 @@ class Options(BaseObject):
     def __init__(
         self,
         connection_string=None,
-        endpoint="https://dc.services.visualstudio.com/v2/track",
         instrumentation_key=None,
         max_batch_size=100,
```

Comment:
Validate `max_batch_size` here to be non-negative?

Reply:
Removed max_batch_size config because we don't support batch workers yet #19